packages/terra-modal-manager/src/ModalManager.jsx

```diff
@@ -5,7 +5,7 @@ import Modal from 'terra-modal';
 
 import AppDelegate from 'terra-app-delegate';
 import SlideGroup from 'terra-slide-group';
-import getBreakpoints from 'terra-responsive-element/lib/breakpoints';
+import breakpoints from 'terra-responsive-element/lib/breakpoints.scss';
```

Comment:
Does this make breakpoints not themeable?

Reply:
They can be themed at build time

Reply:
They can be themed at build time

Reply:
They can be themed at build time

Reply:
Correct, we can pass in a file to override variables or override them directly similar to how we set themes in the sass-loader.

Reply:
Media queries can't actually be themed with CSS custom properties anyway.